Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-categories/ContestCategories.module.scss

```diff
@@ -29,16 +29,46 @@
     transition: opacity 0.2s ease-in-out;
   }
 
+  .categoryListItem {
+    border-bottom: 1px solid colors.$dark-text-color;
+
+    .isChild {
+      border-bottom: none;
+    }
+
+    .categoryListItemContent {
+      font-size: variables.$f-size-16;
+      padding: spacings.$sp-8 spacings.$sp-4;
+
+      &:hover {
+        background-color: #76a2d2;
+        border-radius: 0;
+      }
+    }
+  }
+
+  // categories with no children that are links and load the contests
   .categoryItem {
     align-items: center;
+    border-bottom: 1px solid colors.$light-text-color;
+    border-radius: 0;
+    color: colors.$light-text-color;
     cursor: pointer;
     display: flex;
     font-size: variables.$f-size-16;
-    font-weight: font-weights.$font-weight-regular;
     justify-content: space-between;
     padding: spacings.$sp-12 spacings.$sp-4;
     transition: background-color 0.2s ease-in-out;
 
+    &.darkCategoryItem {
+      border-color: colors.$dark-text-color;
+      color: colors.$dark-text-color;
+    }
+
+    &.mainTreeCategoryItem {
+      padding-left: 28px;
```

Comment:
use spacings

Reply:
We don't have the exact spacing for this case